### PR TITLE
ref(seer): separate out all seer connections

### PR DESCRIPTION
**Pull request by trillville**
there are now 4 different URLs corresponding to the 4 different seer deployments/services

- breakpoint detection
- severity
- grouping
- autofix

i have left anomaly detection for now as the endpoint file is still there, but in a separate PR we can clean that up too

### CODE REVIEW

File: src/sentry/conf/server.py [hunk 1]
```diff
@@ -3711,15 +3711,25 @@ def build_cdc_postgres_init_db_volume(settings: Any) -> dict[str, dict[str, str]
...
-SEVERITY_DETECTION_RETRIES = 1
+SEER_BREAKPOINT_DETECTION_URL = SEER_DEFAULT_URL  # for local development, these share a URL
+SEER_BREAKPOINT_DETECTION_TIMEOUT = 5
+
+SEER_SEVERITY_URL = SEER_DEFAULT_URL  # for local development, these share a URL
+SEER_SEVERITY_TIMEOUT = 0.3  # 300 milliseconds
+SEER_SEVERITY_RETRIES = 1
+
+SEER_AUTOFIX_URL = SEER_DEFAULT_URL  # for local development, these share a URL
+SEER_AUTOFIX_TIMEOUT = 5
```
Comment (jennmueng) [inline src/sentry/conf/server.py:3725]: Is this being used?

File: src/sentry/api/endpoints/organization_transaction_anomaly_detection.py [hunk 2]
```diff
@@ -16,7 +16,7 @@
 from sentry.utils import json
 
 ads_connection_pool = connection_from_url(
```
Comment (jennmueng) [inline src/sentry/api/endpoints/organization_transaction_anomaly_detection.py:18]: you forgot to put `SEER_ANOMALY_DETECTION_TIMEOUT` here

**Review by jennmueng — APPROVED** [review]

File: src/sentry/conf/server.py (hunk 1 above)
Comment (trillville) [reply src/sentry/conf/server.py:3725]: no, ill remove

**Review by jangjodi — APPROVED** [review]

File: src/sentry/conf/server.py [hunk 3]
```diff
@@ -3711,15 +3711,24 @@ def build_cdc_postgres_init_db_volume(settings: Any) -> dict[str, dict[str, str]
...
+SEER_SEVERITY_TIMEOUT = 0.3  # 300 milliseconds
+SEER_SEVERITY_RETRIES = 1
+
+SEER_AUTOFIX_URL = SEER_DEFAULT_URL  # for local development, these share a URL
+
+SEER_GROUPING_URL = SEER_DEFAULT_URL  # for local development, these share a URL
+SEER_GROUPING_TIMEOUT = 1
+
+SEER_ANOMALY_DETECTION_URL = SEER_DEFAULT_URL  # for local development, these share a URL
+SEER_ANOMALY_DETECTION_TIMEOUT = 5
```
Comment (wedamija) [inline src/sentry/conf/server.py:3730]: Should this be 30 to match what it was before?

**Review by wedamija — APPROVED** [review]
Have we overridden any of these in getsentry? We should update the values there first if so

File: src/sentry/conf/server.py (hunk 3 above)
Comment (trillville) [reply src/sentry/conf/server.py:3730]: i think 5 is a better value

**Comment by trillville** [comment]
> Have we overridden any of these in getsentry? We should update the values there first if so

agreed, @dmajere is handling the getsentry/ops side of things, so this PR will just sit here until he's ready

**Pull request merged**